wordpress-mobile/WordPress-Android
# Issue/8098 notifs comment images

Fixes #8098 

Fixing #7849 introduced a new bug. Calling `toString()` on a `Spannable` removes all spans, therefore images (ImageSpans) in comment notifications are not being displayed.

This PR also adds support for animated gif images. Just these 
``` 
WPCustomImageGetter imageGetter = new WPCustomImageGetter(textView,
context.getResources().getDimensionPixelSize(R.dimen.notifications_max_image_size));
```
two lines are related to the gif support, so I thought it would be easier to include it here than to create a separate PR. However, let me know if you prefer to review two separate PRs.

I also removed WPImageGetter as it was its last usage.

To test:
1. Make sure fix from #7849 is still working properly
--------
1. Create a comment on the web with an animated gif image 
- I had to add a second administrator to my site. I wasn't able to add an image as a regular user.
2. Go to the app -> Notifications
3. Click on the comment notification
4. Observe the image is displayed and it's being animated

Note
Anyone can review this PR.
